tools/lint

```diff
@@ -92,6 +92,11 @@ def run():
                                   description="Checks commit messages for common formatting errors."
                                   "(config: .gitlint)")
 
+    sgrep_command = ["sgrep-lint", "--config=./tools/sgrep.yml", "."]
+    linter_config.external_linter('sgrep', sgrep_command,
+                                  description="Syntactic Grep (sgrep) Code Search Tool "
+                                  "(config: ./tools/sgrep.yml")
```

Comment:
```suggestion
    sgrep_command = ["sgrep-lint", "--config=./tools/sgrep.yml", "--error"]
    linter_config.external_linter('sgrep-py', [*sgrep_command, "--lang=python"], ['py']
                                  description="Syntactic Grep (sgrep) Code Search Tool "
                                  "(config: ./tools/sgrep.yml")
```

We want to use the lint system’s existing understanding of which files should be checked and what language they are, instead of scanning the directory again. We also need to exit with nonzero status on errors.

Is there a way to suppress the message “`running 2 rules from 1 config tools/sgrep.yml`” without also suppressing the output on errors?

Reply:
If you/we were to add JavaScript patterns, would a new `linter_config.external_linter` call be introduced with something like `sgrep-js`.

Reply:
Yes, exactly.